mmgen/core/evaluation/metrics.py

```diff
@@ -1030,6 +1029,8 @@ def pil_resize(self, x):
             torch.FloatTensor: Resized tensor.
         """
         if not self._pil_resize_warned:
+            import sys
+            sys.stdout.write('\n')
```

Comment:
Why add this line? You should note that this code will be run by 8 times if you are using distributed training?